src/mono/wasm/build/WasmApp.targets

```diff
@@ -86,7 +81,11 @@
       </_AotInputAssemblies>
 
       <_WasmAssemblies Remove="@(_WasmAssemblies)" />
+
+      <_WasmAOTSearchPaths Include="$(MicrosoftNetCoreAppRuntimePackRidDir)\native" />
+      <_WasmAOTSearchPaths Include="$(MicrosoftNetCoreAppRuntimePackRidDir)\lib\net6.0" />
```

Comment:
Are you able to use `$(TargetFramework)` here instead? Avoids having to keep this file updated between major releases.

Reply:
`$(TargetFramework)` can be different from what the runtime pack has, right? I'm not sure how I can get the one to be used with the pack.

Reply:
Not sure I follow. The TFM for the runtime pack is determined by the project's TFM (by the SDK). I haven't seen a case where they would be different.